src/main/java/com/denizenscript/depenizen/bukkit/clientizen/Channels.java

```diff
@@ -2,6 +2,8 @@
 
 public class Channels {
     public static final String CHANNEL_NAMESPACE = "clientizen";
-    public static final String SET_SCRIPTS = CHANNEL_NAMESPACE + ":" + "set_scripts";
-    public static final String RECEIVE_CONFIRM = CHANNEL_NAMESPACE + ":" + "receive_confirmation";
+    public static final String SET_SCRIPTS = CHANNEL_NAMESPACE + ':' + "set_scripts";
+    public static final String RECEIVE_CONFIRM = CHANNEL_NAMESPACE + ':' + "receive_confirmation";
+    public static final String EVENT_DATA = CHANNEL_NAMESPACE + ':' + "event_data";
+    public static final String RECEIVE_EVENT = CHANNEL_NAMESPACE + ':' + "fire_event";
 }
```

Comment:
Not sure what's going on with these strings, feel like creating them should be a method like in Clientizen

Reply:
True, added a `Channels#id` method